microcosm_flask/swagger/schemas.py

```diff
@@ -65,9 +65,15 @@ def iter_fields(self, schema: Schema) -> Iterable[Tuple[str, Field]]:
         Generates: name, field pairs
 
         """
+        serialized_names = {}
+        for field_name, field_def in schema.declared_fields.items():
+            if isinstance(field_def, Field) and field_def.data_key:
+                serialized_names[field_name] = field_def.data_key
```

Comment:
Full disclosure: I stole this straight from: https://www.programcreek.com/python/example/106766/marshmallow.fields.Field